### PR TITLE
[13.0][MIG] sale_partner_incoterm

### PR DESCRIPTION
**Pull request by jimhoefnagels**

### CODE REVIEW

**Review by lfreeke — APPROVED** [review]
Functional test :+1:

**Review by hparfr — APPROVED** [review]
LGTM

**Comment by jimhoefnagels** [comment]
Hi @OCA/crm-sales-marketing-maintainers,
Could this PR be merged?
Thanks

**Review by jeroenev — APPROVED** [review]

File: sale_partner_incoterm/tests/test_sale_partner_incoterm.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    def test_sale_partner_incoterm(self):
+        """
+        Check that the customer's default incoterm is retrieved in the
+        sales order's onchange
+        """
+        customer = self.env["res.partner"].search([("customer_rank", ">", 1)], limit=1)
+        incoterm = self.env["account.incoterms"].search([], limit=1)
+        customer.write({"sale_incoterm_id": incoterm.id})
+        sale_order = self.env["sale.order"].create({"partner_id": customer.id})
+        sale_order.onchange_partner_id()
```
Comment (simahawk) [inline sale_partner_incoterm/tests/test_sale_partner_incoterm.py:17]: to test the onchanges it's better to use the form class https://github.com/odoo/odoo/blob/13.0/odoo/tests/common.py#L1336
No a blocker, but keep it mind of the next time ;)

**Comment by jimhoefnagels** [comment]
@simahawk  
Rebased with 13.0 and ran the new pre-commit.  Merge shoudl be green now.

**Pull request merged**